src/box/lua/schema.lua

```diff
@@ -1948,6 +1948,10 @@ base_index_mt.select_ffi = function(index, key, opts)
     check_index_arg(index, 'select')
     local ibuf = cord_ibuf_take()
     local key, key_end = tuple_encode(ibuf, key)
+    if not key then
+        cord_ibuf_put(ibuf)
+        return box.error()
+    end
```

Comment:
AFAIU you don't need this, because tuple_encode never returns nil - it raises an exception on error.

Reply:
@drewdzzz Please reply to the comment after addressing it so that reviewers can see what changed. Something like "Fixed" is good enough.